projects/museumpassmusees/reference/partner-api.json

```diff
@@ -430,9 +430,188 @@
+        "security": [
+          {
+            "CLIENT_ACCESS_TOKEN": []
+          }
+        ],
+        "parameters": [],
+        "tags": [
+          "External cards"
+        ]
+      },
+      "parameters": [
+        {
+          "schema": {
+            "type": "string",
+            "enum": [
+              "uitpas",
+              "akaart",
+              "article27"
+            ]
+          },
+          "name": "type",
+          "in": "path",
+          "required": true,
+          "description": "The type of external card"
+        },
+        {
+          "schema": {
+            "type": "string"
+          },
+          "name": "identification",
+          "in": "path",
+          "required": true,
+          "description": "The identification (e.g. card number) of the external card"
+        }
+      ]
+    },
+    "/external-cards/{type}/{identification}/visits": {
+      "post": {
+        "summary": "Register a museum visit for an external card",
+        "operationId": "post-external-cards-type-identification-visits",
+        "responses": {
+          "201": {
+            "description": "External card visit registered."
+          },
+          "401": {
+            "$ref": "#/components/responses/Unauthorized"
+          },
+          "403": {
+            "$ref": "#/components/responses/Forbidden"
+          },
+          "404": {
+            "description": "Not Found. Possible error types:\n\n* https://api.publiq.be/probs/url/not-found",
+            "content": {
+              "application/problem+json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Error"
+                }
+              }
+            }
+          },
+          "503": {
+            "description": "Service Unavailable. Possible error types:\n\n* https://api.publiq.be/probs/museumpass/external-service-unavailable\n\nThe `Retry-After` HTTP Header might include a hint when a retry of this request can be attempted.",
+            "content": {
+              "application/problem+json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Error"
+                }
+              }
+            }
+          }
+        },
+        "description": "Registers a visit of an axternal card at your museum (determined by the client access token used).\n\nTo check if this card is allowed to visit, look up their information using [`GET /external-cards/{type}/{identification}`](./partner-api.json/paths/~1get-external-cards-type-identification~1{type}~1{identification}/get). The response to that request will indicate if the passholder may visit your museum.\n\nIf the external card may not visit your museum and you still try to register a visit for them, the server will return a `403 Forbidden` error response.\n\n## Permissions\n\nThe caller of this request must have the `external card visit registrar` role.",
```

Comment:
Since it's uncommon for POST requests to not have a body, it might be good to mention explicitly in the description that it a body is not required. Otherwise it might seem like we forgot to document it.